src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -71,7 +71,7 @@ class IntelliJPlugin implements Plugin<Project> {
 
         if (JavaPlugin.hasProperty('COMPILE_ONLY_CONFIGURATION_NAME')) {
             project.configurations.getByName(JavaPlugin.COMPILE_ONLY_CONFIGURATION_NAME).extendsFrom idea, ideaPlugins
-            project.configurations.getByName(JavaPlugin.TEST_COMPILE_ONLY_CONFIGURATION_NAME).extendsFrom idea, ideaPlugins
+            project.configurations.getByName(JavaPlugin.TEST_COMPILE_CONFIGURATION_NAME).extendsFrom idea, ideaPlugins
```

Comment:
Oh, I see. Then let's keep `idea` in classpath and exclude `ideaPlugins` from it.

Test Fixture loads all plugins from `idea.plugins.path` property and each plugin have its own ClassLoader to load classes from a corresponding directory, so having them in classpath look redundant at least.